doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst

```diff
@@ -135,60 +288,45 @@ There are two downsides to the implementation above for C functions:
 2.  To follow current implementation standards, documentation should be attached
     directly to the Python source code.
 
-Alternatively for C functions, the implementation of ``like=`` could be moved
-into the C implementation itself. This is not the primary suggestion here due
-to its inherent complexity which would be difficult too long to describe in its
-entirety here, and too tedious for the reader. However, we leave that as an
-option open for discussion.
-
-Usage
------
+The first version of this proposal suggested the implementation above as one
+viable solution for NumPy functions implemented in C. However, due to the
+downsides pointed above we have decided to discard any changes on the Python
+side and resolve those issues with a pure-C implementation . Please refer to
```

Comment:
```suggestion
side and resolve those issues with a pure-C implementation. Please refer to
```

Reply:
Done in https://github.com/numpy/numpy/pull/17093/commits/b6f2c16e23be6fdd38ee89233d13ceecd595c948 .